pkg/couchdb/encoding.go

```diff
@@ -49,3 +49,29 @@ func (vr *ViewRequest) Values() (url.Values, error) {
 
 	return v, nil
 }
+
+// Values transforms a AllDocsRequest into a query string suitable for couchdb
+// ie, where non-scalar fields have been JSON+URL encoded.
+func (adr *AllDocsRequest) Values() (url.Values, error) {
```

Comment:
I don't understand why you have to introduce this method. We already use requests with startkey and endkey at several places in the stack, and I don't see what is different in your case.

Reply:
From what I saw, the startkey and endkey are used for regular views but never for _all_docs, which use a specific `AllDocsRequest` struct https://github.com/cozy/cozy-stack/blob/master/pkg/couchdb/bulk.go#L15
Therefore I needed to redefine the Values() method for this kind of request.